apps/E2E/src/ButtonV1/specs/ButtonV1.spec.android.ts

```diff
@@ -0,0 +1,66 @@
+import NavigateAppPage from '../../common/NavigateAppPage';
+import { PAGE_TIMEOUT, BOOT_APP_TIMEOUT, AndroidAttribute, ANDROID_BUTTON } from '../../common/consts';
+import ButtonV1PageObject from '../pages/ButtonV1PageObject';
+import { BUTTON_TEST_COMPONENT } from '../../ButtonLegacy/consts';
+
+// Before testing begins, allow up to 60 seconds for app to open
+describe('Button Testing Initialization', function () {
+  it('Wait for app load', async () => {
+    await NavigateAppPage.waitForPageDisplayed(BOOT_APP_TIMEOUT);
+    await expect(await NavigateAppPage.isPageLoaded()).toBeTruthy();
+  });
+
+  it('Click and navigate to Button test page', async () => {
+    await ButtonV1PageObject.mobileScrollToComponentButton();
+    await ButtonV1PageObject.waitForButtonDisplayed(PAGE_TIMEOUT);
+
+    /* Click on component button to navigate to test page */
+    await NavigateAppPage.clickAndGoToButtonPage();
+    await ButtonV1PageObject.waitForPageDisplayed(PAGE_TIMEOUT);
+
+    await expect(await ButtonV1PageObject.isPageLoaded()).toBeTruthy();
+  });
+});
+
+describe('ButtonV1 Accessibility Testing', async () => {
+  beforeEach(async () => {
+    await ButtonV1PageObject.mobileScrollToTestElement();
+  });
+
+  it('ButtonV1 - Verify accessibilityLabel', async () => {
+    await expect(
+      await ButtonV1PageObject.compareAttribute(
+        ButtonV1PageObject._primaryComponent,
+        AndroidAttribute.AccessibilityLabel,
+        BUTTON_TEST_COMPONENT,
+      ),
+    ).toBeTrue();
```

Comment:
Let's align with the rest of the file and use `.toBeTruthy()` here. Consistency across APIs. `.toBeTruthy()` also allows us to pass in an error message param

Reply:
Done. Also replaced occurrences of toBeTrue() in other files